src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -213,6 +218,35 @@ export const getApplicationBuilder = () => {
     },
   }));
 
+  mainDi.override(lensFetchInjectable, (di) => {
+    return async (pathnameAndQuery, init) => {
+      const handleLensRequest = di.inject(handleLensRequestInjectable);
+      const { Headers, Response } = di.inject(nodeFetchModuleInjectable);
```

Comment:
It's pretty weird to inject constructor-functions, even more so as here they are specific to foreign, non-contract technical details. 

It would make more sense to inject `factory-functions`, if not instances, when required.

Reply:
The only reason `nodeFetchModuleInjectable` exists at all is because we cannot just import it normally.